docs/pages/distribution/publishing-websites.mdx

```diff
@@ -83,6 +83,8 @@ Open [`http://localhost:5000`](http://localhost:5000) to see your project in act
 
 ## Hosting on third-party services
 
+When deploying from a remote git repository remember to remove the dist directory from the .gitignore file and commit and push the changes. This directory should be selected for the web deployment.
```

Comment:
```suggestion
```

This might not be true for all the deployment services. for example, in the Vercel one, the `outputDirectory ` is mentioned in the vercel config file. And also, when using older SDKs, Expo webpack generates `web-build` as a directory and not `dist`.

So removing this here. Feel free to create a new PR, if you think updates made to amplify are applicable to other deployment services